src/Coduo/PHPHumanizer/Number/Spell.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Coduo\PHPHumanizer\Number;
+
+use Coduo\PHPHumanizer\Translator\Builder;
+
+// Credit to Karl Rixon: http://www.karlrixon.co.uk/writing/convert-numbers-to-words-with-php/
+class Spell
+{
+    /**
+     * @var array
+     */
+    private $map = array(
+        0                   => 'zero',
+        1                   => 'one',
+        2                   => 'two',
+        3                   => 'three',
+        4                   => 'four',
+        5                   => 'five',
+        6                   => 'six',
+        7                   => 'seven',
+        8                   => 'eight',
+        9                   => 'nine',
+        10                  => 'ten',
+        11                  => 'eleven',
+        12                  => 'twelve',
+        13                  => 'thirteen',
+        14                  => 'fourteen',
+        15                  => 'fifteen',
+        16                  => 'sixteen',
+        17                  => 'seventeen',
+        18                  => 'eighteen',
+        19                  => 'nineteen',
+        20                  => 'twenty',
+        30                  => 'thirty',
+        40                  => 'fourty',
+        50                  => 'fifty',
+        60                  => 'sixty',
+        70                  => 'seventy',
+        80                  => 'eighty',
+        90                  => 'ninety',
+        100                 => 'hundred',
+        1000                => 'thousand',
+        1000000             => 'million',
+        1000000000          => 'billion',
+        1000000000000       => 'trillion',
+        1000000000000000    => 'quadrillion',
+        1000000000000000000 => 'quintillion'
+    );
+
+    /**
+     * @param $number
+     * @param $locale
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function __construct($number, $locale = 'en')
```

Comment:
We can move ``$number, $locale = 'en'`` into ``convert`` function and pass ``Translator`` instead of ``Builder`` via constructor. 

This will give us something like that: 
```
class NumberSpeller
{
    public function __construct(Translator $translator)
    {} 

    public function spell($number, $locale = 'en')
    {}
}
```